LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini

```diff
@@ -1,9 +1,5 @@
-; This file covers the Research, engineering and stats of all weapons
+[LW_WeaponPack.X2StrategyElement_LaserTechs]
```

Comment:
Mod name is LWOTC_WeaponPack this won't add the ini files correctly. You'll need to correct this across the entire file.